packages/main/src/Popup.js

```diff
@@ -18,36 +19,14 @@ const metadata = {
 	slots: /** @lends  sap.ui.webcomponents.main.Popup.prototype */ {
 
 		/**
-		 * Defines the content of the Web Component.
+		 * Defines the content of the Popup.
```

Comment:
Really minor, but this "Popup" text would appear in the Dialog API as well. If we consider the term Popup good for the Dialog it's ok, otherwise we should fix it.